glean-core/ios/GleanTests/Debug/GleanDebugUtilityTests.swift

```diff
@@ -25,17 +25,17 @@ class GleanDebugUtilityTests: XCTestCase {
         // regex for ping tag names.
         var url = URL(string: "test://glean?tagPings=This-tag-is-not-valid-because-it-is-too-long")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertNil(Glean.shared.configuration?.pingTag)
+        XCTAssertEqual(Glean.shared.testDebugViewTag(), "")
 
         url = URL(string: "test://glean?tagPings=Invalid_Chars@!!$")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertNil(Glean.shared.configuration?.pingTag)
+        XCTAssertEqual(Glean.shared.testDebugViewTag(), "")
 
         // Check valid tag: This should update the pingTag value of the
         // configuration object.
         url = URL(string: "test://glean?tagPings=Glean-Ping")
         Glean.shared.handleCustomUrl(url: url!)
-        XCTAssertEqual("Glean-Ping", Glean.shared.configuration?.pingTag)
+        XCTAssertEqual("Glean-Ping", Glean.shared.testDebugViewTag())
```

Comment:
Maybe there's a way to test this by looking at the HTTP headers -- or, honestly, I'd be ok with just removing this test case.

Reply:
I removed this and added a test to the rust upload module about the X-Debug-ID header just to be sure.